.travis.yml

```diff
@@ -16,4 +16,5 @@ script:
   - export PATH=$PATH:$GOPATH/bin
   - make travis
 after_script:
-  - find *.log | xargs -I{} sh -c "cat {} | nc termbin.com 9999 | xargs -r0 printf '{} uploaded to %s'"
+  - echo "Uploading to termbin.com..." && find *.log | xargs -I{} sh -c "cat {} | nc termbin.com 9999 | xargs -r0 printf '{} uploaded to %s'"
+  - echo "Uploading to file.io..." && find *.log | xargs -I{} sh -c "curl -s -F 'file=@{}' https://file.io | xargs -r0 printf '{} uploaded, result is %s\n'"
```

Comment:
Yeah so rather than uploading all the _individual_ files, it seems better to just go with the original gzip approach.